webapp/store/ui/record/actions.js

```diff
@@ -9,15 +9,14 @@ import * as NodeRefData from '@core/record/nodeRefData'
 
 import { debounceAction } from '@webapp/utils/reduxUtils'
 
+import { AppSavingActions } from '@webapp/store/app'
 import { SurveyState } from '@webapp/store/survey'
-
-
+import { UserState } from '@webapp/store/user'
 import { LoaderActions, NotificationActions } from '@webapp/store/ui'
-import { AppSavingActions } from '@webapp/store/app'
 
```

Comment:
would it be relatively quick to split Actions into separate files (create, read, update, delete) ?

Reply:
right